docs/manual/docs/overview/authors.md

```diff
@@ -1,54 +1,84 @@
+
+GeoNetwork activity determined over six months:
+
+* GN4: [is:pr is:closed  reviewed-by:@me](https://github.com/geonetwork/core-geonetwork/pulls?q=is%3Apr+is%3Aclosed++reviewed-by%3A%40me)
+* GN5: [is:pr is:closed  reviewed-by:@me](https://github.com/geonetwork/geonetwork/pulls?q=is%3Apr+is%3Aclosed++reviewed-by%3A%40me)
+
+```bash
+gh pr list --repo geonetwork/core-geonetwork --state closed --search "reviewed-by:@me"
+gh pr list --repo geonetwork/geonetwork --state closed --search "reviewed-by:@me"
+```
+
+This is primarily an active maintenance role, based on trust given the level of responsibility required. If you are interested in participating please speak up on the developer forum and start by testing and reviewing pull-requests. Nominated committers demonstrate contributions and dedication to the project.
+
+Earning community trust as an committer allows you to merge a pull-request once your review is complete, although this does come with increased leadership responsibility.
+
+We ask that committers remain active, meeting responsibilities to review pull requests, address security vulnerabilities and vote on roadmap proposals. As maintainers we understand that life can get in the way of community particpation – you are welcome to rejoin as your availability permits you to meet the responsibilities.
+
+Reference:
+
+- [Active Committers](https://github.com/orgs/geonetwork/teams/active-committers) (GitHub Team)
+
+## Commmitters
+
+GeoNetwork welcomes contributors to take a more active role in the project. If you have capacity please work towards building your comfort level with the code base and express an interest in taking a greater role.
+
+Having commit access is required to volunteer for such activities as making a release.
+
+Reference:
+
+* [Committers](https://github.com/orgs/geonetwork/teams/committers) (GitHub Team)
 
 ## Contributors
 
-Contributors list is available [here](https://github.com/geonetwork/core-geonetwork/graphs/contributors).
+GeoNetwork uses a pull-request workflow to review and accept changes. Pull-requests must be submitted against the main branch first, and may be back ported as required.
+
+For larger changes we ask that a proposal be submitted to the roadmap for discussion.
+
+Reference:
+
+- [Contributors](https://github.com/geonetwork/core-geonetwork/graphs/contributors) (GitHub Insights)
+- [CONTRIBUTING.md](https://github.com/geonetwork/core-geonetwork/blob/main/CONTRIBUTING.md) (GitHub Repository)
+- [Contributing Guide](../contributing/index.md) (GeoNetwork Documentation)
+
+## Thanks
 
-## The documentation
+We would like to thank prior contributors, committers, and leadership for their role in making GeoNetwork a success:
 
-The documentation was written by the GeoNetwork opensource developers and other community members.
+* Andrea Carboni - Leadership
+* Antonio Cerciello
+* Jo Cook - Leadership
+* Jesse Eichar - Leadership
+* Michel Gabriel
+* Florent Gravin - Leadership
+* Olivier Guyot
```

Comment:
```suggestion
```

Reply:
oh non I didn't remove myself from the leadership group :) just the "thanks" paragraph at the bottom